dominicfallows/df-multichannel-app
# Update site

- Update site with latest bio
- Update dependencies
- Add Lanzarote Dashboard  - see webcams and info about Lanzarote, in place of actually getting to visit there this year!
